src/compas/numerical/mma/mma_numpy.py

```diff
@@ -2,17 +2,849 @@
 from __future__ import division
 from __future__ import print_function
 
+from scipy.sparse import diags
+from scipy.linalg import solve
+import numpy as np
+import logging
+import os
+import time
+
+
+"""
+Orginal work Written by Krister Svanberg in Matlab. This is the python version of the code written
+by Arjen Deetman.
```

Comment:
Do you refer to this code? https://github.com/arjendeetman/GCMMA-MMA-Python
If you used that as a reference, we cannot include it because it's under a GPL license, that is not compatible with our MIT license.

Reply:
Hi Gonzalo.
Thanks for the feedback. I will see what I can do and reopen this pull request in the future.
Thanks.